routes/authenticate.js

```diff
@@ -0,0 +1,27 @@
+const Router = require('express').Router;
+const User = require('../models/User');
+const jwt = require('jsonwebtoken');
+
+const router = Router();
+
+router.post('/', (req, res) => {
+  User.authenticate(req.body.userName, req.body.password, (err, user) => {
+    if (err) {
+      res.status(400).json(err);
+      return;
+    }
+
+    const payload = {
```

Comment:
Producing the user token should be provided by a function.  Wrap this logic so that we can do something like:
`res.cookie('token', getAuthToken(user))`